Documentation/ApiOverview/Database/QueryBuilder/Index.rst

```diff
@@ -853,7 +853,12 @@ Remarks:
     out the first n records". Internally, :sql:`LIMIT` will be added by
     Doctrine DBAL and set to a very high value.
 
-*   Use :php:`->setMaxResults(null)` to retrieve all results.
+*   In TYPO3 v13 it will be possible to use :php:`->setMaxResults(null)` to retrieve 
+    all results.
+    
+*   In older Versions up to TYPO3 v12 it is possible to use :php:`->setMaxResults(0)` 
```

Comment:
```suggestion
*   In versions before TYPO3 v13 it is possible to use :php:`->setMaxResults(0)` 
```

Reply:
Yes, you're right. It's better to document v12 first and then only mention the changed behaviour in v13.